src/encord_active/app/page_menu.py

```diff
@@ -29,7 +29,7 @@
     "Data Drift": project_similarity(),
 }
 
-DEFAULT_PAGE_PATH = ["Data Quality", "Summary"]
+DEFAULT_PAGE_PATH = ["Model Quality", "Explorer"]
```

Comment:
Think this was for testing efficiency purposes no?

Reply:
Correct, removed